frontend/src/static/css/v_summary.scss

```diff
@@ -96,7 +96,7 @@
       vertical-align: middle;
     }
 
-    &--repo {
+    &--group {
```

Comment:
Will `group-name` be better ? Will be more specific I feel, and easier for future developers to understand ?

Reply:
agree, if its a name component then it should be `groupname`, otherwise may get misinterpreted as other things e.g. group container